FSAE.X/CAN.h

```diff
@@ -11,8 +11,7 @@
  * CAN message IDs
  */
 #define ERROR_ID        0x00F
-#define ANALOG_REAR_ID  0x050
-#define ANALOG_FRONT_ID 0x060
+#define SPM_ID          0x050
```

Comment:
I think we should make the SPM messages send out on 0x300 or something. The most critical messages on the bus are ECU messages (used by the PDM to actively change load states and monitor error conditions) and GCM messages (used for error conditions and for the driver's display). so these should have the highest priority. The SPM messages will mostly be used for data logging, I think, so it should be fine to reduce their priority.